lib/snippets/assert-npm-version-tag

```diff
@@ -11,9 +11,10 @@ if [ $? != '0' ]; then
 fi
 
 TAG_REV=`git rev-list -n1 v$VERSION`
-HEAD_REV=`git rev-parse HEAD`
+HEAD_REV=`git rev-list -n1 HEAD`
+HEAD_REV_WITHOUT_MERGES=`git rev-list -n1 --no-merges HEAD`
 
-if [ $TAG_REV != $HEAD_REV ]; then
+if [ $TAG_REV != $HEAD_REV ] || [ $TAG_REV != $HEAD_REV_WITHOUT_MERGES ]; then
```

Comment:
I'm trying to recall what happened yesterday, could be wrong. But, I don't think this will actually solve the issue you had yesterday. `$TAG_REV != $HEAD_REV` will still yield true and error out, no?